src/MagnumPlugins/AssimpImporter/AssimpImporter.h

```diff
@@ -287,9 +296,6 @@ verbosity levels in each instance.
 -   Custom mesh attributes (such as `object_id` in Stanford PLY files) are
     not imported.
 -   Texture coordinate layers with other than two components are skipped
--   For some file formats (such as COLLADA), Assimp may create a dummy
-    "skeleton visualizer" mesh if the file has no mesh data. For others (such
-    as glTF) not.
```

Comment:
This is fixed by setting `AI_CONFIG_IMPORT_NO_SKELETON_MESHES`, but there's another comment down below in the *Scene import* section ("[...] there's sometimes just a single root node") which I'm not sure about. The node collapsing doesn't happen anymore with this setting (the test got adapted accordingly), but I wonder if this is the only circumstance where this happens.

Reply:
See [my comment here](https://github.com/mosra/magnum-plugins/pull/99#discussion_r663461312).